### PR TITLE
NUI-914 Handle transparency

### PR DESCRIPTION
**Pull request by tmathern**
## Description

To be merged after https://github.com/adobe/asset-compute-sdk/pull/70. 
We noticed that support for handling transparency during post-processing for some formats was missing. 

This is to add transparency handling support for images as follow:
- Input format and output format both support transparency (e.g. PNG, GIF, BMP, TIF): keep transparency.
- Input format supports transparency, but output format doesn't (e.g. input is PNG, output is JPEG): The rendition gets a white background.
- Input format does not support transparency, but output format does (e.g. input is JPEG, output is PNG): no logic change (it is not possible to choose a color that would become transparent).
- Neither input nor output format support transparency: no logic change.

Fixes  [NUI-914](https://jira.corp.adobe.com/browse/NUI-914).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: lib/postprocessing/image.js [hunk 1]
```diff
@@ -39,17 +39,28 @@ const SUPPORTED_OUTPUT_FMT = {
...
 
+const FORMAT_SUPPORTS_TRANSPARENCY = {
+    "png": true,
+    "tif": true,
+    "tiff": true,
+    "gif": true,
+    "bmp": true,
+    "svg": true
+};
+const DEFAULT_BACKGROUND_COLOR = "#FFFFFF";
```
Comment (DrGirlfriend) [inline lib/postprocessing/image.js:62]: Probably should change this from `#FFFFFF` to `rgba(0, 0, 0, 0.0)`

Comment (tmathern) [reply lib/postprocessing/image.js:62]: `rgba(0, 0, 0, 0.0)` failed during testing for me. Any special way to add quotes around it that I may be missing?

Comment (tmathern) [reply lib/postprocessing/image.js:62]: Nevermind! It's changed.

**Review by DrGirlfriend — APPROVED** [review]
LGTM

**Pull request merged**